src/tools/rustdoc-js/tester.js

```diff
@@ -190,6 +190,30 @@ function loadThings(thingsToLoad, kindOfLoad, funcToCall, fileContent) {
     return content;
 }
 
+function contentToDiffLine(key, value) {
+    return `"${key}": "${value}",`;
+}
+
+// This function is only called when no matching result was found and therefore will only display
+// the diff with the first item.
+function betterLookingDiff(entry, data) {
+    let output = '{\n';
+    data = data[0];
```

Comment:
Maybe it's better to pass in the first element, instead of passing in the whole array but only using the first one?

Reply:
I made something completely different first where we had a more "global" diff. The result wasn't great so I dropped it but maybe we will give it another try in the future. So in this case, I think it's better to keep it this way. What do you think?

Reply:
Hmm, can you say more about why it wasn't great? I still think it makes sense to show all the differences instead of just the first.

Reply:
Because generally you have one item in the tests whereas the search will return you up to a few hundreds. It wouldn't make sense to make the diff between all of them. However, you made me realize something so I'll improve it once more! :D

Reply:
Hum no, not gonna work either haha.

Reply:
Can you print the diff only for items that changed, instead of printing all of them unconditionally?

Reply:
You call this function in case you didn't find a matching item. How do I know which is the correct one then? :)

Reply:
That's also why I made the small improvement in case order matters, so that you get the first item that you should have gotten.